src/auth/jwt-auth.guard.ts

```diff
@@ -0,0 +1,4 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+@Injectable()
+export class JwtAuthGuard extends AuthGuard('jwt') {}
```

Comment:
PullProofAI

Consider adding error handling to the `JwtAuthGuard` class to handle authentication failures gracefully.

```typescript
import { Injectable, UnauthorizedException } from '@nestjs/common';
import { AuthGuard } from '@nestjs/passport';

@Injectable()
export class JwtAuthGuard extends AuthGuard('jwt') {
  handleRequest(err, user, info) {
    if (err || !user) {
      throw err || new UnauthorizedException();
    }
    return user;
  }
}
```



<!-- This is an auto-generated comment by PullProofAI -->